apps/user/src/components/form/Calculators/GradePreview/GradePreview.tsx

```diff
@@ -1,9 +1,20 @@
 import { color, font } from '@maru/theme';
-import { Row, Th, Td } from '@maru/ui';
+import { Row, Td, Th } from '@maru/ui';
 import styled from 'styled-components';
+import {
+    useAttendanceScore,
+    useCertificateScore,
+    useGradeScore,
+    useVolunteerScore,
+} from './GradePreview.hooks';
 
 // 모의 성적 계산
 const GradePreview = () => {
+    const { regularScore, specialScore } = useGradeScore();
+    const { score: attendanceScore } = useAttendanceScore();
+    const { score: volunteerScore } = useVolunteerScore();
+    const { score: certificateScore } = useCertificateScore();
```

Comment:
```suggestion
    const { regularScore, specialScore } = useGradeScore();
    const { attendanceScore } = useAttendanceScore();
    const { volunteerScore } = useVolunteerScore();
    const { certificateScore } = useCertificateScore();
```

넘길때부터 네이밍을 이렇게 지어보면 어떨까요?